### DISCUSSION TITLE
Recommendation of Stable Parameters

### DISCUSSION THREAD

**Discussion by justuswolff-audit** [optuna/optuna#5387] [Q&A] [answered]
Hi, I honestly love optuna and I've been using it a lot lately. But I ran into an issue (there might already exist a solution that I am not aware of) and that is what happens if the hyperparameters of the best run are unstable?
Let me set the scene: you run a lot of trials, one trial was the best, you chose those hyperparameters for your model training and it failed... Why? Because that was just a lucky trial and when looking at the vizualisations you could actually see that setting one of the parameters from the best trial produced also a lot of really bad objectives. Now my question is: is there a way to get recommendations for hyperparameters rather than using the best trial's ones? Something like confidence intervals of the best hyperparmeters?

**Comment by nzw0301** [comment, marked as answer]
I'm not sure but the Incoming FAQ item might be helpful https://github.com/optuna/optuna/pull/5339.

**Reply by justuswolff-audit** [reply]
Thank you very much, that is exactly what I was looking for!
